tests/integration/test_pr_agent_config_validation.py

```
@@ -0,0 +1,418 @@
+import yaml
+from pathlib import Path
+
+
+class TestPRAgentConfigStructure:
+    """Test basic structure of PR agent configuration."""
+    
+    @pytest.fixture
+    def pr_agent_config(self):
+        """Load the PR agent configuration file."""
+        config_path = Path('.github/pr-agent-config.yml')
+        with open(config_path, 'r') as f:
```

Ruff (0.14.6)

Unnecessary mode argument

Remove mode argument

(UP015)